conf/application.conf

```diff
@@ -26,17 +26,6 @@ play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
 
 appName = catalogue-frontend
 
-# Session configuration
-# ~~~~~
-application.session.httpOnly = false
-
-application.session.secure = false
```

Comment:
these have not been valid - play supports `play.http.session.secure` with deprecated alternative `session.secure`